doc/release-notes.md

```diff
@@ -26,63 +26,81 @@ using version < 0.13 you will have to reindex (start with -reindex-chainstate
+
+Signed binaries for Windows
+---------------------------
+This release is the first one to include signed binaries for Windows.
+
+New RPC command: quorum memberof <proTxHash>
+--------------------------------------------
+This RPC allows you to verify which quorums a masternode is supposed to be a member of. It will also show
+if the masternode succesfully participated in the DKG process.
+
+More information about number of InstantSend locks
+--------------------------------------------------
+The debug console will now show how many InstantSend locks Dash Core knows about. Please note that this number
+does not necessarily equal the number of mempool transactions.
+
+The "getmempoolinfo" RPC also has a new field now which shows the same information.
+
+0.14.0.3 Change log
+===================
 
-Fixed out-of-sync masternode list UI
-------------------------------------
-The masternode tab, which shows the masternode list, was not always up-to-date as it missed some internal
-updates. This should be fixed now.
+TODO
 
-0.14.0.2 Change log
-===================
+See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.14.0.2...dashpay:v0.14.0.3).
 
-See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.14.0.1...dashpay:v0.14.0.2).
-
-- [`d2ff63e8d`](https://github.com/dashpay/dash/commit/d2ff63e8d) Use std::unique_ptr for mnList in CSimplifiedMNList (#3014)
-- [`321bbf5af`](https://github.com/dashpay/dash/commit/321bbf5af) Fix excessive memory use when flushing chainstate and EvoDB (#3008)
-- [`0410259dd`](https://github.com/dashpay/dash/commit/0410259dd) Fix 2 common Travis failures which happen when Travis has network issues (#3003)
-- [`8d763c144`](https://github.com/dashpay/dash/commit/8d763c144) Only load signingActiveQuorumCount + 1 quorums into cache (#3002)
-- [`2dc1b06ec`](https://github.com/dashpay/dash/commit/2dc1b06ec) Remove skipped denom from the list on tx commit (#2997)
-- [`dff2c851d`](https://github.com/dashpay/dash/commit/dff2c851d) Update manpages for 0.14.0.2 (#2999)
-- [`46c4f5844`](https://github.com/dashpay/dash/commit/46c4f5844) Use Travis stages instead of custom timeouts (#2948)
-- [`49c37b82a`](https://github.com/dashpay/dash/commit/49c37b82a) Back off for 1m when connecting to quorum masternodes (#2975)
-- [`c1f756fd9`](https://github.com/dashpay/dash/commit/c1f756fd9) Multiple speed optimizations for deterministic MN list handling (#2972)
-- [`11699f540`](https://github.com/dashpay/dash/commit/11699f540) Process/keep messages/connections from PoSe-banned MNs (#2967)
-- [`c5415e746`](https://github.com/dashpay/dash/commit/c5415e746) Fix UI masternode list (#2966)
-- [`fb6f0e04d`](https://github.com/dashpay/dash/commit/fb6f0e04d) Bump version to 0.14.0.2 and copy release notes (#2991)
 
 Credits
 =======
 
 Thanks to everyone who directly contributed to this release:
 
 - Alexander Block (codablock)
+- Nathan Marley (nmarley)
+- strophy
```

Comment:
https://github.com/dashpay/dash/pull/3046/commits/781b165796dc6d7766cf1dc99b6db8943bb5b706
should add PastaPastaPasta :)

Reply:
Whoops, sorry :see_no_evil: